### ISSUE TITLE
Toolbars and menus shouldn't be displayed as web page contents in Vivaldi

### ISSUE THREAD

**Issue by Cleversn** [nvaccess/nvda#12813]
### Steps to reproduce:
1. Download the Vivaldi web browser from https://downloads.vivaldi.com/stable/Vivaldi.4.1.2369.21.x64.exe
2. Install it and open any web page you like;
### Actual behavior:
NVDA displays toolbars, menus and other elements from the browser's interface, right at the beginning of the page.
### Expected behavior:
NVDA shouldn't do it, but behave as in e.g. Firefox and Edge.
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
2021.1
#### Windows version:
Windows 10 Pro 21H1
#### Name and version of other software in use when reproducing the issue:
Vivaldi 4.2
#### Does the issue still occur after restarting your computer?
Yes.
#### If add-ons are disabled, is your problem still occurring?
Yes.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

**Comment by Cleversn** [comment]
Just tested the latest Vivaldi, version 5.6 on Windows 11 22 h2 with NVDA 2022.4. The problem stays the same: interface elements being displayed in the virtual buffer, where only the page should be.

I'll be grateful if someone could investigate whether it's easier to solve this at the NVDA side, or I should contact Vivaldi.

**Comment by Adriani90** [comment]
I guess this is an interface related issue that is fully in hands of the vivaldi developers. This is probably how they tried to design the user experience. I think NVDA does not have the ability to specify what's included in the virtual buffer, this is the browser itself if I am not mistaken. cc: @jcsteh

**Comment by Adriani90** [comment]
This issue is related to a design choice the app or web developers need to make, and it is nothing wrong done by NVDA in this case. I am closing as invalid.
